packages/tools/devtools/devtools-example/src/FluidObject.ts

```diff
@@ -175,42 +175,98 @@ export class AppData extends DataObject {
 
 	private populateSharedTree(sharedTree: ITree): void {
 		// Set up SharedTree for visualization
-		const builder = new SchemaFactory("DefaultVisualizer_SharedTree_Test");
+		const builder = new SchemaFactory("devtools-example");
```

Comment:
Once you have everything here working as expected, I would recommend refactoring this to be more like a real example domain. Maybe something like a contact list? We could also save this for a future task.

Reply:
If we want to wait for a separate PR to simplify this, that's fine. But in the immediate term, we may want to revert the changes here. Your unit tests should ensure the behaviors you were testing here don't regress.